docs/DeveloperGuide.md

```diff
@@ -4,30 +4,148 @@
 
 {list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well}
 
+* [JSON-java](https://github.com/stleary/JSON-java)
+* [three-ten-extra](https://www.threeten.org/threeten-extra/)
+* * Requesting and Parsing of data from API into Java
+* https://www.youtube.com/watch?v=lDEfoSwyYFg
 ## Design & implementation
-
 {Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
```

Comment:
Perhaps this should be replaced with something along the lines of "This section describes the design and implementation of the product, with UML diagrams and code snippets where applicable." instead.